apps/sms/views/conversation/js/conversation.js

```diff
@@ -1193,7 +1187,10 @@ var ConversationView = {
       return threadPromise.then((id) => {
         return Navigation.toPanel('thread', { id: id, focusComposer: true });
       }, () => {
-        return Navigation.toPanel('composer', { activity: parameters });
+        return Navigation.toPanel('composer', {
+          activity: parameters,
+          focusComposer: !!(parameters && parameters.number)
```

Comment:
basically the same logic than https://github.com/mozilla-b2g/gaia/pull/31055/files#diff-33b9f27b0cd1246a89a6096fa1769c95L731